### ISSUE TITLE
IAM custom names

### ISSUE THREAD

**Issue by tingletech** [hybox/aws#74]
Looks like only one hybox stack can be run at a time per account?

<img width="1008" alt="screen shot 2016-08-13 at 5 40 01 pm" src="https://cloud.githubusercontent.com/assets/227374/17646418/02a26eb2-617d-11e6-9a43-ccffba44abc1.png">

```
templates/application.json
331:            "PolicyName": "S3Access"

templates/codepipeline.json
134:            "PolicyName": "codepipeline-service",

templates/mail.json
396:          "PolicyName": "root",

templates/slack.json
155:          "PolicyName": "root",

templates/travis.json
67:            "PolicyName": "S3Access"

templates/worker.json
412:          "PolicyName": "root",

templates/zookeeper.json
321:            "PolicyName": "exhibitor-s3",
```

I searched all these names in IAM, and I don't have any of these names, so I guessed I'm good to go.

**Comment by cbeer** [comment]
We certainly expected only one stack per account, and our understanding is that practice is somewhat common in AWS-land. That said, I don't foresee any problem prefixing the top-level stack name as part of it.

**Comment by tingletech** [comment]
One of the major pain points in getting this to run is how long it takes to delete a stack after things fail.  I had to uncheck "rollback on failure" or the error messages would disappear.  I can't try again till the old stack finally decides to give up the ghost, which now seems to take several hours before getting to DELETE_FAILED.  I made the mistake of trying to start a stack before the old stack had fully deleted, and maybe I've got things messed up now from that.  Maybe it is related to this.
